perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -54,6 +80,107 @@
                   'num_machines': 1}
 
 
+JOB_FILE_TEMPLATE = """
```

Comment:
@stfeng @noahl The default behavior of the new code is doing random RW with varying queue depth, while the default behavior of the old code is doing seq RW, ran RW, which is more like a overview of the disk performance.
If user is using their own fio.job, this PR won't make any difference. However, if someone write some automation using the default old fio benchmark, the published metric will definitely be different and we might break their automation.
I suggest, we should at least mention in CHANGES.md that this is a breaking change that changes the default behavior of the fio benchmark, or go through the community, which is far more complex, since this benchmark is in standard_set.